mm2src/coins/z_coin.rs

```diff
@@ -1410,6 +1472,10 @@ impl SwapOps for ZCoin {
         utxo_common::extract_secret(secret_hash, spend_tx)
     }
 
+    fn check_tx_signed_by_pub(&self, _tx: &[u8], _expected_pub: &[u8]) -> Result<bool, MmError<ValidatePaymentError>> {
+        unimplemented!();
+    }
+
```

Comment:
Again, please don't move functions or variables around if it's unrelated to this PR, it makes the review process harder.